elastic/kibana
# [App Search] Set up Analytics router

## Summary

This PR sets up the Analytics router/sub-pages - it does not add any functioning views.

I recommend following along by commit if possible. Also a review by either one of you (@byronhulcher or @JasonStoltz) would be excellent, I think this is small/straightforward enough that we likely don't need both.

## QA

- Required setup - Enterprise Search w/ sample engine
- For each of the below pages, confirm you see the correct: 1) TODO text 2) breadcrumb text 3) Analytics link in left sidebar is properly highlighted
- [x] http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/analytics
- [x] http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/analytics/top_queries
- [x] http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/analytics/top_queries_no_results
- [x] http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/analytics/top_queries_no_clicks
- [x] http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/analytics/top_queries_with_clicks
- [x] http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/analytics/recent_queries
- Other pages to test:
- [x] http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/analytics/query_detail/test (should not have working breadcrumbs just yet)
- [x] http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/analytics/404 (should show a 404 view)

### Checklist
